adapters/postgres/postgres.go

```diff
@@ -42,22 +42,43 @@ func chkInvalidIdentifier(identifer string) bool {
 func WhereByRequest(r *http.Request, initialPlaceholderID int) (whereSyntax string, values []interface{}, err error) {
 	whereKey := []string{}
 	whereValues := []string{}
+	var value string
+	// set default operation
+	op := "="
```

Comment:
the  default operator must be `$eq` not `=` because you always call `GetOperator` and it expects receive `$eq` no `=`